jobs/workflow_test.groovy

```diff
@@ -129,20 +130,28 @@ import utilities.StatusUpdater
       if (isPR) { // update commit with pending status while tests run
         shell StatusUpdater.updateStatus(
           commitStatus: 'pending', repoName: '${COMPONENT_REPO}', commitSHA: '${ACTUAL_COMMIT}', description: 'Running e2e tests...')
+
+        setupHelmcEnv = new File("${WORKSPACE}/bash/scripts/setup_helmc_environment.sh").text
+        setupHelmcEnv += """
+          mkdir -p ${defaults.tmpPath}
+          setup-helmc-env >> ${defaults.envFile}
```

Comment:
It doesn't strictly need to be since we see here this is the first time we are writing to the file -- I've just been in the habit of appending by default since the file is located in a JOB_NAME/BUILD_NUMBER-specific location, just to be 'safe' as there can be many times where we write to it.

That being said, I can certainly change to `>` for correctness.